src/buffer.rs

```diff
@@ -1340,6 +1344,14 @@ pub(crate) type Gray16Image = ImageBuffer<Luma<u16>, Vec<u16>>;
 /// Sendable 16-bit grayscale + alpha channel image buffer
 pub(crate) type GrayAlpha16Image = ImageBuffer<LumaA<u16>, Vec<u16>>;
 
+/// An image buffer for 32-bit float RGB pixels,
+/// where the backing container is a flattened vector of floats.
+pub(crate) type Rgb32FImage = ImageBuffer<Rgb<f32>, Vec<f32>>;
```

Comment:
Why are these `pub(crate)` and not `pub`? I would like them public to reduce the typing required by users

Reply:
We hadn't yet decided if they were ready. I think we don't have any test coverage on the conversion to/from the other color types for example. As a pure container though they should be fine.

Reply:
I only thought about is as a container. I'll make them public :)